examples-gallery/plot_non_continuous_parameter_estimation.py

```diff
@@ -0,0 +1,209 @@
+# %%
+"""
+
+Parameter Identification from Non-Continuous Measurements.
```

Comment:
You didn't agree that "contiguous" was the correct word?

contiguous /kən-tĭg′yoo͞-əs/
adjective

    Sharing an edge or boundary; touching.
    Neighboring; adjacent.
    Connecting without a break.

Reply:
I guess it can be either, but we have continuous measurements, they are just not contiguous in time.

Reply:
> You didn't agree that "contiguous" was the correct word?
> 
> contiguous /kən-tĭg′yoo͞-əs/ adjective
> 
> ```
> Sharing an edge or boundary; touching.
> Neighboring; adjacent.
> Connecting without a break. 
> ```

:-)  :-)
I thought is was a typo. Knowing you I should have known better! Will change and push.

Reply:
It doesn't' really matter, I just get nit picky on wording. I'm working an journal article this morning doing the final editing and my mind is in the mode of being extremely precise with wording.

Reply:
> It doesn't' really matter, I just get nit picky on wording. I'm working an journal article this morning doing the final editing and my mind is in the mode of being extremely precise with wording.

Can I get a copy when you are done with it?

Reply:
Sure.

Reply:
Thanks!